service/src/main/java/uk/gov/hmcts/reform/fpl/NotifyTemplates.java

```diff
@@ -14,10 +14,11 @@ private NotifyTemplates() {
     public static final String ORDER_NOTIFICATION_TEMPLATE = "1f7c134e-f9c0-44ba-aa50-fce53eb208f7";
     public static final String CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE =
         "64cc7ea5-ca74-4679-912f-6d84d1c7e4cd";
-    public static final String CMO_ORDER_ISSUED_DOCUMENT_LINK_NOTIFICATION_TEMPLATE  =
+    public static final String CMO_ORDER_ISSUED_DOCUMENT_LINK_NOTIFICATION_TEMPLATE =
         "931dcd86-d331-4273-abaa-208daed0761a";
     public static final String CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE =
         "6bdbc939-14bb-4dae-ac35-e0d381840719";
     public static final String CMO_REJECTED_BY_JUDGE_TEMPLATE = "44124d30-dce0-4ae4-8e74-27dc43fc282a";
-    public static final String NOTICE_OF_PLACEMENT_ORDER_UPLOADED_TEMPLATE = "ac0a0091-0b2e-44b9-afa1-321590b5d984";
+    public static final String PLACEMENT_APPLICATION_NOTIFICATION_TEMPLATE = "ac0a0091-0b2e-44b9-afa1-321590b5d984";
```

Comment:
Hmm not your's of course but I wonder if we should take the time now to name this better now that there's many placement notification templates.

Reply:
I can sneak in - what do you think, NEW_PLACEMENT....